packages/bridge/src/nomic/index.ts

```diff
@@ -228,4 +471,19 @@ export class NomicBridgeProvider implements BridgeProvider {
       return [];
     }
   }
+
+  async getProtoRegistry() {
+    if (!this.protoRegistry) {
+      const [{ ibcProtoRegistry, osmosisProtoRegistry }, { Registry }] =
+        await Promise.all([
```

Comment:
Is this valuable on the server? More of a Q vs a request. Does it slightly reduce cold start times by doing something like this?

Reply:
I'm not entirely sure. I'll have to investigate